src/device-api/v1.coffee

```diff
@@ -47,16 +46,14 @@ exports.createV1Api = (router, applications) ->
 					return service
 			.then (service) ->
 				res.status(200).json({ containerId: service.containerId })
-		.catch (err) ->
-			res.status(503).send(err?.message or err or 'Unknown error')
+		.catch(next)
 
-	router.post '/v1/apps/:appId/stop', (req, res) ->
-		v1StopOrStart(req, res, 'stop')
+	createV1StopOrStartHandler = (action) -> _.partial(v1StopOrStart, _, _, _, action)
 
-	router.post '/v1/apps/:appId/start', (req, res) ->
-		v1StopOrStart(req, res, 'start')
+	router.post '/v1/apps/:appId/stop', createV1StopOrStartHandler('stop')
```

Comment:
Please add parens for single line functions calls in coffeescript

Reply:
Done. But why doesn't our linter require this?

Reply:
You can either enforce parens in all cases or none @roman-mazur, what we decided on for coffeescript was the more nuanced "always for functions without a callback/single line callback, optional/never for functions with a multiline callback" - overall that makes the best use of significant whitespace in coffeescript whilst avoid a lot of the confusion around things like `a b c, d, e, f (g h), (i j)` where things can get crazy with various interactions of parens and whitespace and it's hard to parse without really thinking about it (that example compiles to `a(b(c, d, e, f(g(h), i(j))))`)
Either way we're steadily moving to typescript which removes the coffeescript linting capabilities as an issue :)